installers/Installer.hs

```diff
@@ -1,18 +1,20 @@
-import           Data.Text as T
-import           Data.Monoid ((<>))
-import           System.Info
-import           Turtle
-import           Turtle.Line          (unsafeTextToLine)
+{-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
Isn't this the default extension now?

Reply:
I had doubts about removing these pragmas because I am not familiar with how `nix-build` treats `cabal` files (whether it knows about them at all), and also because `RewriteLibs.hs` was clearly intended to be run as a standalone script. I've removed `NoImplicitPrelude` from all files except for `RewriteLibs.hs` now.